frontend/src/util/ws_mousetrap_fork.js

```diff
@@ -1066,3 +1066,35 @@
         });
     }
 })(typeof window !== 'undefined' ? window : null, typeof window !== 'undefined' ? document : null);
+
+/**
+ * adds a pause and unpause method to Mousetrap
+ * this allows you to enable or disable keyboard shortcuts
+ * without having to reset Mousetrap and rebind everything
+ */
+/* global Mousetrap:true */
+(function(Mousetrap) {
```

Comment:
Did you write this code yourself or was it automatically added somehow?

Reply:
I added it so we could perform keyboard shortcuts albeit having the upload menu.

Reply:
@Levilian it would be good if you moved this code to a separate file. Everything else in this file is vendor code from mousetrap, right?